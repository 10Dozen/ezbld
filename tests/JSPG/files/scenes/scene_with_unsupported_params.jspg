# SceneWithUnsupportedParams | title

Desc line 1.

*

*exec: { Foo.Bar() }

**

Desc line 5.

*Desc line 6.

**Desc line 7.
