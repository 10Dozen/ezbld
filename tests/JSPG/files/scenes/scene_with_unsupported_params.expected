[0]
cls = JSPGScene
name = SceneWithUnsupportedParams
type = title
desc =
    Desc line 1.

    *

    *portrait: picture.jpg

    **

    Desc line 5.

    *Desc line 6.

    **Desc line 7.
